### PR TITLE
correct text

### PR DESCRIPTION
**Pull request by AlexKVal**
its -> it's -> it is
a HTTP -> we read 'H' as 'eich' -> an HTTP

### CODE REVIEW

File: doc/api/cluster.markdown [hunk 1]
```diff
@@ -24,7 +24,7 @@ all share server ports.
       });
     } else {
       // Workers can share any TCP connection
-      // In this case its a HTTP server
+      // In this case it is an HTTP server
```
Comment (misterdjules) [inline doc/api/cluster.markdown]: @joyent/node-coreteam Could a native English speaker please confirm the usage of "an" instead of "a" here? Thank you!

Comment (tresf) [inline doc/api/cluster.markdown]: Confirmed per BBC:

> Take the eighth letter of the alphabet, pronounce it haitch and then look for the slightly agonised look in some people's eyes. One suggestion is that it touches on a long anxiety in English over the letter aitch. In the 19th Century, it was normal to pronounce hospital, hotel and herb without the h.Oct 28, 2010

So "an HTTP" should be similar to "an hour". :)

**Comment by jasnell** [comment]
Will land this in a separate commit and PR in http://github.com/nodejs/node master

**Comment by jasnell** [comment]
Closing here. New PR opened.

**Pull request closed without merging**